docker_cli/_docker_mamba.py

```diff
@@ -0,0 +1,235 @@
+from __future__ import annotations
+
+import os
+import shlex
+import textwrap
+from typing import Iterable, Optional, Tuple, overload
+
+
+def mamba_install_dockerfile(
+    env_specfile: os.PathLike[str] | str = "spec-file.txt",
```

Comment:
I'd suggest removing the default value here (and in similar functions in `_docker_mamba.py`.

As we [decided previously](https://github.com/isce3-testing/image-sandbox/pull/2#discussion_r1230160965), the command line `setup conda runtime` and `setup conda dev` commands will use the lockfiles located in the `env_files` directory by default, rather than assuming that the user has a file named `spec-file.txt` in their current directory.

The functions here in `_docker_mamba.py` are intended to be more low-level and generic, so I'm not sure that there's any particular default value that makes sense here.

```suggestion
    env_specfile: os.PathLike[str] | str,
```